pyaedt/edb.py

```diff
@@ -1356,36 +1356,123 @@ def solve_siwave(self):
         return True
 
     @pyaedt_function_handler()
-    def add_design_variable(self, variable_name, variable_value):
+    def add_design_variable(self, variable_name, variable_value, is_parameter=False):
         """Add a design variable.
 
         Parameters
         ----------
         variable_name : str
-            Name of the variable.
+            Name of the variable. To be added as a project variable, the name must begin with ``$``.
         variable_value : str, float
             Value of the variable with units.
+        is_parameter : bool, optional
+            The default is ``True`` to add the variable as a parameter default.
+            If ``False``, the variable is added as a local variable.
 
         Returns
         -------
         tuple
-            tuple containing the ``AddVariable`` result and variable server.
+            Tuple containing the ``AddVariable`` result and variable server.
+
+        Examples
+        --------
+
+        >>> from pyaedt import Edb
+        >>> edb_app = Edb()
+        >>> boolean_1, ant_length = edb_app.add_design_variable("my_local_variable", "1cm")
+        >>> boolean_2, para_length = edb_app.change_design_variable_value("my_parameter", "1m", is_parameter=True
+        >>> boolean_3, project_length = edb_app.change_design_variable_value("$my_project_variable", "1m")
+
+
         """
-        is_parameter = True
         if "$" in variable_name:
-            var_server = self.db.GetVariableServer()
-            is_parameter = False
+            if variable_name.index("$") == 0:
+                var_server = self.db.GetVariableServer()
+                is_parameter = False
+                string_message = ["Creating project variable %s.", "Project variable %s exists. You can use it."]
```

Comment:
@jeremich41 I truly think all of these strings would be enhanced by including "already" in the sentence about about it existing. For example, in this line: Project variable %s already exists. You can use it.